### PR TITLE
DOCSUP-9599: Edit and translate to Russian (LDAP)

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation (edit and translate to Russian)

### CODE REVIEW

**Comment by sevirov** [comment]
@vitlibar Hi. Please, review this PR.

**Comment by sevirov** [comment]
@traceon Hi. Please, review this PR.

File: docs/ru/operations/external-authenticators/ldap.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 <yandex>
     <!- ... -->
     <ldap_servers>
+	    <!- Typical LDAP server. -->
```
Comment (traceon) [inline docs/ru/operations/external-authenticators/ldap.md:20]:
```suggestion
	    <!- Typical LDAP server. -->
```

**Review by traceon — APPROVED** [review]
LGTM, however bind DN -> "DN привязки" and other places where bind (LDAP-specific terminology) is used, looks kind of artificial. Is that how typical LDAP things are translated to Russian?

File: docs/ru/operations/external-authenticators/ldap.md (hunk 1 above)
Comment (traceon) [reply docs/ru/operations/external-authenticators/ldap.md:20]: Tab to spaces.

**Comment by sevirov** [comment]
@traceon Hi. I found this translation in several sources (bind DN -> DN подключения). I fixed it. Please, review again.

**Comment by traceon** [comment]
@sevirov thanks, looks great.

**Pull request merged**